test/powershell/Language/Classes/scripting.enums.tests.ps1

```diff
@@ -86,6 +80,15 @@ Describe 'Basic enum errors' -Tags "CI" {
         Remove-Module LanguageTestSupport
     }
 
+    Context 'Enum definition' {
+        It "Enum values definition have space-insensitive syntax" {
+            $errors0 = Get-ParseResults -Src 'enum E0 {e=0}'
+            $errors1 = Get-ParseResults -Src 'enum E0 {e =0}'
+            $errors0.Count | Should Be 0
+            $errors1.Count | Should Be 0

```

Comment:
We don't need to *hide* the script we want to test in a string. If the file fails to parse, we will see the error.

I asked that you validate the value being initialized like we do in earlier tests.

In fact, you could delete a single space on line 72, add a comment saying "don't delete this space", and I'd be happy that you've covered everything.